crates/router/src/connector/stripe/transformers.rs

```diff
@@ -332,24 +390,12 @@ pub struct BankTransferData {
     pub email: Email,
 }
 
-#[derive(Debug, Eq, PartialEq, Serialize)]
-pub struct StripeAchSourceRequest {
-    #[serde(rename = "type")]
-    pub transfer_type: StripePaymentMethodType,
-    #[serde(rename = "owner[email]")]
-    pub email: Email,
-    pub currency: String,
-}
-
 #[derive(Debug, Eq, PartialEq, Serialize)]
 #[serde(untagged)]
```

Comment:
Add a comment which says, when de-serialise is added then untagged should be removed.